Thrift.podspec

```diff
@@ -1,18 +1,18 @@
 Pod::Spec.new do |s|
   s.name          = "Thrift"
-  s.version       = "1.0.0"
+  s.version       = "0.12.0
   s.summary       = "Apache Thrift is a lightweight, language-independent software stack with an associated code generation mechanism for RPC."
   s.description   = <<-DESC
 The Apache Thrift software framework, for scalable cross-language services development, combines a software stack with a code generation engine to build services that work efficiently and seamlessly between C++, Java, Python, PHP, Ruby, Erlang, Perl, Haskell, C#, Cocoa, JavaScript, Node.js, Smalltalk, OCaml and Delphi and other languages.
                     DESC
   s.homepage      = "http://thrift.apache.org"
-  s.license       = { :type => 'Apache License, Version 2.0', :url => 'https://raw.github.com/apache/thrift/thrift-0.9.0/LICENSE' }
+  s.license       = { :type => 'Apache License, Version 2.0', :url => 'https://raw.github.com/apache/thrift/thrift-0.12.0/LICENSE' }
```

Comment:
The old link does not work.  Try `https://raw.githubusercontent.com/apache/thrift/0.12.0/LICENSE`

Reply:
Also remember to modify master with `https://raw.githubusercontent.com/apache/thrift/1.0.0/LICENSE`

Reply:
Fixed locally.  Using the standard apache license link.